CODE/Sources/PennyBOT/Handlers/InteractionHandler.swift

```diff
@@ -706,3 +881,72 @@ extension Payloads.InteractionResponse.Modal: Response {
     /// Responses containing a modal can't be an edit to another message.
```

Comment:
I've also recently introduced this `Response` type. The command-responding function can return any type that conforms to `Response`, then the top-level send-message function would automatically know what to send where.
Didn't turn out to be the cleanest  approach, but works for now.